### PR TITLE
use excerpt in rss if no meta_description is set

### PR DESCRIPTION
**Pull request by rambii**
issue #9210

- added fallback for post excerpt
- added test

### CODE REVIEW

**Comment by rambii** [comment]
How is the build error related to the commit?

**Comment by AileenCGN** [comment]
@rambii was just a timeout issue. I restarted the test. It's passing now 🎉

**Comment by rambii** [comment]
@AileenCGN thanks for the feedback :)

File: core/server/data/xml/rss/index.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ generateFeed = function generateFeed(data) {
             htmlContent = processUrls(post.html, data.siteUrl, itemUrl),
             item = {
                 title: post.title,
-                description: post.meta_description || downsize(htmlContent.html(), {words: 50}),
+                description: post.meta_description || post.custom_excerpt || downsize(htmlContent.html(), {words: 50}),
```
Comment (kevinansfield) [inline core/server/data/xml/rss/index.js]: @rambii thanks for looking at this! One quick change that is required... the `custom_excerpt` should be used if available, then fall back to the others:

`description: post.custom_excerpt || post.meta_description || downsize(htmlContent.html(), {words: 50}),`

**Review by kevinansfield — CHANGES_REQUESTED** [review]

**Comment by rambii** [comment]
@kevinansfield I updated it accordingly (y)

**Comment by AileenCGN** [comment]
@kevinansfield This looks good to me now! Can I merge?

**Review by kevinansfield — APPROVED** [review]

**Pull request merged**